src/components/IndividualWorkContainer/ProgressBar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Stepper, Step, StepLabel } from "@material-ui/core";
+
+const ProgressBar = () => {
+  const steps = [
+    "step1",
+    "step2",
+    "step3",
+    "step4",
+    "step5",
+    "step6",
+    "step7",
+    "step8"
+  ];
+  const currentStep = 4;
+
```

Comment:
`steps` and `currentStep` are meant to be passed as props to the progress container. If you need to mock them with hard-coded values, put such values in `IndividualWorkContainer`